### ISSUE TITLE
The split of the Stanford Cars dataset.

### ISSUE THREAD

**Issue by ppanzx** [KevinMusgrave/pytorch-metric-learning#685]
Thank you for your commendable efforts in your work. I have a question regarding the split of the Stanford Cars dataset, which comprises 16,185 images representing 196 car models.

In most metric-learning literature, the dataset split is described as follows: "The first 98 classes (8,054 images) are used for training, and the remaining 98 classes (8,131 images) are held out for testing."

However, the split mentioned in the [Torchvision](https://pytorch.org/vision/stable/_modules/torchvision/datasets/stanford_cars.html#StanfordCars) documentation states that "The data is split into 8,144 training images and 8,041 testing images, with an approximately 50-50 split for each class.", the training and testing split of which is different from current metric-learning community.

Unfortunately, the [official website](https://ai.stanford.edu/~jkrause/cars/car_dataset.html) is currently inaccessible, leaving me uncertain about the specific split used in this implementation.

Could you kindly provide me with a detailed split list (rather than the raw images) used in your implementation of the Stanford Cars dataset?

Thank you for your attention to this matter.

**Comment by KevinMusgrave** [comment]
This library doesn't have any dataset classes. If you're referring to the [Metric Learning Reality Check](https://arxiv.org/abs/2003.08505) paper, you could follow its description of the train/val/test splits:

```
- The first half of classes are used for cross validation, and the 4 partitions
are created deterministically: the first 0-12.5% of classes make up the first
partition, the next 12.5-25% of classes make up the second partition, and so
on. The training set comprises 3 of the 4 partitions, and cycles through all
leave-one-out possibilities. As a result, the training and validation sets are
always class-disjoint, so optimizing for validation set performance should be
a good proxy for accuracy on open-set tasks. Training stops when validation
accuracy plateaus.

– The second half of classes are used as the test set. This is the same setting
that metric learning papers have used for years, and we use it so that results
can be compared more easily to past papers.
```

You could take a look at [the code for the paper](https://github.com/KevinMusgrave/powerful-benchmarker/tree/metric-learning) but it's quite gnarly and difficult to understand, so I suggest just writing your own split-creation code.
